web/cypress/fixtures/plugin-dataset.json

```diff
@@ -0,0 +1,336 @@
+        "timeout": 1000,
+        "audience": "University",
+        "grant_type": "urn:ietf:params:oauth:grant-type:uma-ticket"
+      }
+    },
+    {
+      "shouldValid": false,
+      "data": {
+        "permissions": ["res:customer#scopes:view"]
+      }
+    },
+    {
+      "shouldValid": true,
+      "data": {
+        "token_endpoint": "http://127.0.0.1:8090/auth/realms/University/protocol/openid-connect/token",
+        "permissions": ["course_resource#view"],
+        "audience": "course_management",
+        "grant_type": "urn:ietf:params:oauth:grant-type:uma-ticket",
+        "timeout": 3000
+      }
+    }
+  ],
+  "batch-requests": [],
+  "consumer-restriction": [],
+  "cors": [],
+  "echo": [],
+  "fault-injection": [],
+  "grpc-transcode": [],
+  "http-logger": [],
+  "ip-restriction": [],
+  "kafka-logger": [],
+  "limit-conn": [
+    {
+      "shouldValid": true,
+      "data": {
+        "conn": 1,
+        "burst": 0,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "remote_addr"
+      }
+    },
+    {
+      "shouldValid": false,
+      "data": {
+        "conn": 1,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "remote_addr"
+      }
+    },
+    {
+      "shouldValid": false,
+      "data": {
+        "burst": 0,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "remote_addr"
+      }
+    },
+    {
+      "shouldValid": false,
+      "data": {
+        "conn": -1,
+        "burst": 0,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "remote_addr"
+      }
+    },
+    {
+      "shouldValid": true,
+      "data": {
+        "conn": 100,
+        "burst": 50,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "server_addr"
+      }
+    },
+    {
+      "shouldValid": true,
+      "data": {
+        "conn": 5,
+        "burst": 1,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "http_x_real_ip"
+      }
+    },
+    {
+      "shouldValid": true,
+      "data": {
+        "conn": 5,
+        "burst": 1,
+        "default_conn_delay": 0.1,
+        "rejected_code": 503,
+        "key": "http_x_forwarded_for"
+      }
+    },
+    {
+      "shouldValid": true,
+      "data": {
+        "conn": 2,
+        "burst": 1,
+        "default_conn_delay": 0.1,
+        "key": "remote_addr"
+      }
+    }
+  ],
+  "limit-count": [],
+  "limit-req": [],
+  "openid-connect": [],
+  "prometheus": [],
+  "proxy-cache": [],
+  "proxy-mirror": [],
+  "proxy-rewrite": [],
+  "redirect": [],
+  "referer-restriction": [],
+  "request-id": [],
+  "response-rewrite": [],
+  "serverless-post-function": [],
+  "serverless-pre-function": [],
+  "sls-logger": [],
+  "syslog": [],
+  "tcp-logger": [],
+  "traffic-split": [],
+  "udp-logger": [],
+  "uri-blocker": [],
+  "zipkin": [],
+  "request-validation": [],
+  "mqtt-proxy": []
+}
```

Comment:
this plugin-dataset is particularly valuable😊, not only for the e2e test, it would be useful for users with some instructions